### ISSUE TITLE
Get Marker Name

### ISSUE THREAD

**Issue by jms5194** [RomeoDespres/reapy#136]
I'm wondering if I'm just missing something- totally possible- but I can't seem to find a way to retrieve the name of a marker. When setting a marker, you can optionally define a name for the marker, but upon retrieving a list of markers- the reapy.marker class does not seem to include a name parameter. And I don't see any other way to request the name of a marker. Is that incorrect? 

Thanks!

**Comment by john-kurkowski** [comment]
You're right, `Project.add_marker` [takes an optional `name`](https://github.com/RomeoDespres/reapy/blob/730627cee6f39fc26d6ebc8a3df0112e5921cd9f/reapy/core/project/project.py#L89), while the `Marker` class has no mention of name. 

As a workaround, you can fetch the marker name directly from the [REAPER API `EnumProjectMarkers2`](https://www.reaper.fm/sdk/reascript/reascripthelp.html#EnumProjectMarkers2) (or `EnumProjectMarkers1` or `EnumProjectMarkers3`) return tuple, like on [this line](https://github.com/RomeoDespres/reapy/blob/730627cee6f39fc26d6ebc8a3df0112e5921cd9f/reapy/core/project/marker.py#L59). Instead of the marker position at index `4`, I think the name is at `6`.

**Comment by Levitanus** [comment]
In `reapy-boost` markers have no bugs and better functional
https://github.com/Levitanus/reapy-boost/blob/master/reapy_boost/core/project/marker.py#L123

**Comment by jms5194** [comment]
Thanks- all.  John- that's actually where I went next- it seems that the reason the name probably wasn't included in reapy is that EnumProjectMarkers1/2/3 doesn't properly include the marker name in the Python implementation. Looks like a long-standing bug in the Reaper code, based on forum reports may date all the way back to 2010. 

https://forum.cockos.com/showthread.php?p=2291096#post2291096

Levitnus- I'll try reapy-boost and see how that works out! Thanks for the help.

**Comment by Levitanus** [comment]
[jms5194](https://github.com/jms5194), I haven't also updated my fork for a couple of years, as moved to rust reaper extensions, but at least I'll manage all pull-requests, that come to the fork. So, If you want to fix something ‒ you're welcome

**Comment by jms5194** [comment]
Hey Levitanus- I'm working on getting reapy-boost working in my project- but it appears that the repaired marker functionality all require SWS to be installed in Reaper- does that seem correct to you? I haven't used SWS a ton- but seeing missing functions like SNM_CreateFastString would seem to be SWS functions, yeah? Thanks!

**Comment by jms5194** [comment]
Answered my own question- does seem to require SWS. Thanks- I'll certainly put in a pull request if I make any useful changes. Thanks!
